crates/sui-framework/packages/sui-framework/sources/test/test_scenario.move

```diff
@@ -103,6 +103,146 @@ public fun begin(sender: address): Scenario {
     }
 }
 
+/// Update the `Scenario` context to use `ids_created` as the number of IDs created
+public fun with_ids_created(
+    scenario: &mut Scenario,
```

Comment:
I'm guessing there's a reason to take `&mut Scenario` to `&mut Scenario` vs `Scenario` to `Scenario`?